plugin/v2/settable.go

```diff
@@ -0,0 +1,73 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
```

Comment:
no not plugin, in this PR name is the name of the env variable.
`docker plugins set DEBUG.value=1`

In the next PR, when I add mounts, you will be able to do
`docker plugin set my_mount.source=$(pwd) my_mount.destination=/work`